.cirrus.yml

```diff
@@ -0,0 +1,63 @@
+env:
+  CIRRUS_CLONE_DEPTH: 1
+
+fedora_30_task:
+  container:
+    image: quay.io/avocado-framework/avocado-vt-ci-fedora-30
```

Comment:
Have you considered creating "avocado-vt-ci-fedora-latest" to avoid the need to update this line every 6-12 months?